contracts/escrow/dapps/CompoundClaimComp.sol

```diff
@@ -48,7 +48,12 @@ contract CompoundClaimCompFacet is PausableMods, DappMods {
             address(COMPTROLLER_ADDRESS_PROVIDER_ADDRESS),
             abi.encodeWithSignature("claimComp(address)", escrow)
         );
-        emit CompoundClaimed(escrow, loanID);
+
+        // Add AAVE to escrow token list
+        // TODO: get the COMP token (add as immutable constructor variable)
+        LibEscrow.tokenUpdated(loanID, address(0));
```

Comment:
this should be throwing an error in the tests when called.. but its not.

this function calls `ERC20(tokenAddress).balanceOf()` which should revert since `address(0)` is not a contract.

can we verify that the tests are working @dumebi

Reply:
This is because we only claim comp (to pass) when we have accrued comp. and during the test, the escrow does not have accrued comp

Reply:
we should then also have a test that advances time so that it can accrue COMP

Reply:
fixing these on `feature/claimAaveCompToUser`